website/docs/sdks/index.md

```diff
@@ -112,6 +112,7 @@ Here's some of the fantastic work our community has done to make Unleash work in
 - [silvercar/unleash-client-kotlin](https://github.com/silvercar/unleash-client-kotlin) (Kotlin)
 - [Stogon/unleash-bundle](https://git.stogon.io/Stogon/unleash-bundle/) (PHP - Symfony)
 - [uekoetter.dev/unleash-client-dart](https://pub.dev/packages/unleash) (Dart)
+- [aruizs/unleash-client-cpp](https://github.com/aruizs/unleash-client-cpp) (C++)
```

Comment:
Sweet! Can you move this to between "AppsFlyer[...]" and "mikefrancis[...]"?

Reply:
Sure! I didn't realize the list follows that order, I applied a FIFO method.

Reply:
No worries! That looks great!